examples/neural_compressor/question-answering/README.md

```diff
@@ -55,6 +55,8 @@ python run_qa.py \
     --output_dir /tmp/squad_output
 ```
 
+The flag `--use_ipex` can be passed along to use INC IPEX backend. Default backend is `pytorch_fx` if value not specified. INC IPEX only supports static quantization for now.
```

Comment:
Do you think we should add an extra comment on why it's interesting to use IPEX ? Also I think we should raise an error if the user tries a combination which is not supported (IPEX + dynamic quantization for example)

Reply:
Explanation and warning added.